sw/device/tests/sim_dv/all_escalation_resets_test.c

```diff
@@ -1003,8 +1008,9 @@ void check_alert_dump() {
     if (i == kExpectedAlertNumber) {
       CHECK(actual_info.alert_cause[i], "Expected alert cause %d to be set", i);
     } else {
-      CHECK(!actual_info.alert_cause[i], "Expected alert cause %d to be clear",
-            i);
+      if (actual_info.alert_cause[i]) {
```

Comment:
There are cases where a fatal alert generates another one, for example, some lc_ctrl faults lead to otp_ctrl faults, so this relaxes the requirement.

Reply:
Nit: maybe put this comment into the source code?

Reply:
Done.